tests/functional/adapter/materialization/test_on_table_exists.py

```diff
@@ -25,19 +41,40 @@ def project_config_update(self):
             },
         }
 
-    # everything that goes in the "seeds" directory
-    @pytest.fixture(scope="class")
-    def seeds(self):
-        return {
-            "seed.csv": seed_csv,
-        }
+    # The actual sequence of dbt commands and assertions
+    # pytest will take care of all "setup" + "teardown"
+    def test_run_seed_test(self, project):
+        # seed seeds
+        results = run_dbt(["seed"], expect_pass=True)
+        assert len(results) == 1
+        # run models two times to check on_table_exists = 'drop'
+        results = run_dbt(["run"], expect_pass=True)
+        assert len(results) == 1
+        results = run_dbt(["run"], expect_pass=True)
+        assert len(results) == 1
+        # test tests
+        results = run_dbt(["test"], expect_pass=True)
+        assert len(results) == 3
+
+        # check if the data was loaded correctly
+        check_relations_equal(project.adapter, ["seed", "materialization"])
+
+
+@pytest.mark.iceberg
```

Comment:
delta not supported yet? 

cc @findinpath

Reply:
AFAIK it is currently supported only on iceberg
https://github.com/trinodb/trino/issues/13180#issuecomment-1828830623

Reply:
> delta not supported yet?

`CREATE OR REPLACE` for Delta will be supported as of this week in Galaxy.

OSS see https://github.com/trinodb/trino/pull/19991

Reply:
`CREATE OR REPLACE` is live on Galaxy and can be used for testing both Iceberg & Delta Lake tables.